corehq/ex-submodules/casexml/apps/phone/tasks.py

```diff
@@ -103,5 +103,7 @@ def prune_synclogs():
             week="%02d" % week
         )
         drop_query = "DROP TABLE IF EXISTS {}".format(table_name)
-        get_cursor(SyncLogSQL).execute(drop_query)
+        db = router.db_for_write(SyncLogSQL)
+        with connections[db].cursor() as cursor:
```

Comment:
Should this look something more like `SyncLogSQL.get_cursor` to match some of the partitioned models?

Reply:
Since this isn't a partitioned model it doesn't need any special treatment